query-engine/query-engine/src/dmmf/schema/ast.rs

```diff
@@ -1,13 +1,13 @@
+use std::collections::HashMap;
+
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Serialize, Deserialize, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct DmmfSchema {
-    pub root_query_type: String,
-    pub root_mutation_type: String,
```

Comment:
Is that information now stored elsewhere? Or do we just rely on convention from now on?

Reply:
Convention is enough for now.